test/e2e/data/e2e_conf.yaml

```diff
@@ -61,7 +67,8 @@ providers:
           - old: --metrics-bind-addr=127.0.0.1:8080
             new: --metrics-bind-addr=:8080
       - name: v1.0.0
-        value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.0.0/core-components.yaml"
+        # FIXME: rollback before merge
```

Comment:
NOTE: All of the FIXME"s in this file will be rolled back once https://github.com/kubernetes-sigs/cluster-api/pull/5583 is merged, but we will have to use nightly artifacts for the latest controller / CRDs